### PR TITLE
JS/Kotlin adding examples and links

### PR DESCRIPTION
**Pull request by fufu930**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

Added JS examples and linked Kotlin examples from a previous PR

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by titusfortner** [comment]
Can you add the JS references in wheel.en files as well? 
When we build the preview for Netlify it should grab from your branch to display properly

Thanks

**Comment by titusfortner** [comment]
Oh shoot, I thought @diemol had dialed this one in. No matter, I pulled it locally to review.

File: examples/javascript/test/actionsApi/wheelTest.spec.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      assert.ok(await inViewport(footer))
+    })
+
+    it('Scroll from an element by a given amount', async function () {
+      await driver.get("https://www.selenium.dev/selenium/web/scrolling_tests/frame_with_nested_scrolling_frame_out_of_view.html")
+
+      const iframe = await driver.findElement(By.css("iframe"))
+
+      await driver.actions()
+        .scroll(0, 0, 0, 300, iframe)
```
Comment (titusfortner) [inline examples/javascript/test/actionsApi/wheelTest.spec.js]: match the other examples; use 200

File: examples/javascript/test/actionsApi/wheelTest.spec.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    })
+
+    it('Scroll from an element with an offset', async function () {
+      await driver.get("https://www.selenium.dev/selenium/web/scrolling_tests/frame_with_nested_scrolling_frame_out_of_view.html")
+
+      const iframe = await driver.findElement(By.css("iframe"))
+      const footer = await driver.findElement(By.css("footer"))
+
+      await driver.actions()
+        .scroll(0, -30, 0, 300, footer)
```
Comment (titusfortner) [inline examples/javascript/test/actionsApi/wheelTest.spec.js]: should be 0, -50, 0, 200 to match other examples

File: examples/javascript/test/actionsApi/wheelTest.spec.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      assert.ok(await inViewport(checkbox))
+    })
+
+    it('Scroll from an offset of origin (element) by given amount', async function () {
+      await driver.get("https://www.selenium.dev/selenium/web/scrolling_tests/frame_with_nested_scrolling_frame_out_of_view.html")
+
+      const iframe = await driver.findElement(By.css("footer"))
+
+      await driver.actions()
+        .scroll(10, 10, 0, 5000)
```
Comment (titusfortner) [inline examples/javascript/test/actionsApi/wheelTest.spec.js]: 10, 10, 0, 200

**Review by titusfortner — CHANGES_REQUESTED** [review]
Thanks for this work!

The tests should work with the values that the rest of the languages are using, please update to those.

File: examples/javascript/test/actionsApi/wheelTest.spec.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+suite(function (env) {
+  describe('Actions API - Wheel Tests', function () {
+    let driver
+
+    before(async function () {
+      driver = await env.builder().build()
+    })
+
+    after(() => driver.quit())
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/wheelTest.spec.js]: after(async() => await driver.quit())

**Review by harsha509 — CHANGES_REQUESTED** [review]

**Review by titusfortner — APPROVED** [review]

**Review by harsha509 — APPROVED** [review]

**Review by harsha509 — APPROVED** [review]

**Comment by harsha509** [comment]
Thank you @hanceanic !

**Pull request merged**